src/entity/src/acl.js

```diff
@@ -24,7 +23,8 @@ export default class Acl {
      * @private
      * @type {Object}
      */
-    this.acl = cloneDeep(entity._acl);
+    entity._acl = entity._acl || {};
+    this.acl = entity._acl;
```

Comment:
@thomasconner the `Acl` class seems to have an `acl` variable. Can we instead just have the properties directly on the instance?